Docs: Remove old Cloud/dashboard and replace with new Cloud/dashboard

joelhans opened the pull request:
<!--
Describe the change in summary section, including rationale and design decisions.
Include "Fixes #nnn" if you are fixing an existing issue.

In "Component Name" section write which component is changed in this PR. This
will help us review your PR quicker.

In "Test Plan" provide enough detail on how you plan to test this PR so that a reviewer can validate your tests. If our CI covers sufficient tests, then state which tests cover the change.

If you have more information you want to add, write them in "Additional
Information" section. This is usually used to help others understand your
motivation behind this change. A step-by-step reproduction of the problem is
helpful if there is no related issue.
-->

##### Summary

This PR strips the documentation strips as many references to the old Netdata Cloud and the old dashboard (both visuals and features) as I could find. References to Nodes View/My nodes menu have been replaced with Cloud/Visited nodes, along with new instructions on the **Sign in** button on every dashboard.

There is certainly a lot of optimization work to be done, but the goal here is, in broad strokes, not present multiple messages about the dashboard or the Agent's link to Cloud.

##### Component Name

docs

##### Test Plan

<!---
Provide enough detail so that your reviewer can understand which test-cases you
have covered, and recreate them if necessary. If sufficient tests are covered
by our CI, then state which tests cover the change.
-->

##### Additional Information

joelhans commented:
@jacekkolasa A lot of this PR is outside your scope, but there are lots of images of the new dashboard, along with descriptions of the new navbars and **Sign in** button. I'd appreciate if you could take a look at anything relevant to your work.

joelhans commented:
I fixed the initial errors that @cakrit had pointed out in the call, but now the Netlify checks fail in a different (but expected and irrelevant way) because I deleted the three Cloud-related docs we had previously:

```
4:34:08 AM: WARNING -  A relative path to 'docs/netdata-cloud/README.md' is included in the 'nav' configuration, which is not found in the documentation files
4:34:08 AM: WARNING -  A relative path to 'docs/netdata-cloud/signing-in.md' is included in the 'nav' configuration, which is not found in the documentation files
4:34:08 AM: WARNING -  A relative path to 'docs/netdata-cloud/nodes-view.md' is included in the 'nav' configuration, which is not found in the documentation files
```

amoss commented on docs/step-by-step/step-02.md:
```diff
@@ -128,11 +117,10 @@ We built Netdata to be a big sandbox for learning more about your systems and ap
 
-![Aniamted GIF of chart
-synchronziation](https://user-images.githubusercontent.com/1153921/63464271-c03a9c00-c413-11e9-971d-245238926193.gif)
+![Animated GIF of the standard Netdata dashboard being manipulated and synchronizing
```
The gif doesn't seem to match the text - it shows resizing a chart rather than panning or zooming. There is another section (and gif) about resizing later - is this one correct?

joelhans replied:
Fixed this. I mixed up all my gifs.

amoss commented on docs/step-by-step/step-03.md:
```diff
@@ -13,160 +14,79 @@ and performance](step-01.md).
-We get it. That's why we built [Netdata Cloud](/docs/netdata-cloud/README.md), which connects many distributed agents
-together for a seamless experience when monitoring multiple systems.
+We get it. That's why we built [Netdata Cloud](https://learn.netdata.cloud/docs/cloud/), which connects many distributed
+agents for a seamless experience when monitoring an entire infrastructure of Agent-monitored nodes.
```
Should this be "Netdata-monitored" nodes? i.e. our product rather than one of the components.

amoss commented on docs/step-by-step/step-03.md:
```diff
@@ -13,160 +14,79 @@ and performance](step-01.md).
 
-![Animated GIF of the various view
-modes](https://user-images.githubusercontent.com/1153921/65068318-4bd51900-d93d-11e9-8720-b3bd76809d16.gif)
+To add nodes to your Visited nodes menu, you first need to navigate to that node's dashboard, then click the **Sign in**
```
The capital Vs look jarring and out of place. Should all of these references be "Visitied Nodes" -> possibly put it in italics to make it more obvious that it is being quoted?

joelhans replied:
I ended up rephrasing how I write about this menu, not referencing the name of the menu as much. I think it's a different way to address your valid icky feeling.

joelhans commented:
@jacekkolasa @amoss Thanks guys. I fixed those few text comments, replaced the wrong gif, and added new ones for the positive/negative feature and the resize feature.
